cmake/ccf_app.cmake

```diff
@@ -30,7 +30,7 @@ if((NOT ${IS_VALID_TARGET}))
 endif()
 
 # Find OpenEnclave package
-find_package(OpenEnclave 0.16.1 CONFIG REQUIRED)
+find_package(OpenEnclave 0.13 CONFIG REQUIRED)
```

Comment:
I assume this is in error? We are not rolling back to OE 0.13.

Reply:
yes definitely in error. Will rollback.

Reply:
I need to rebuild my machine :(